### ISSUE TITLE
kmodel output does not change

### ISSUE THREAD

**Issue by MoritzKrauth** [kendryte/nncase#206]
I am trying to use a modified version of the MobileNetV2 model of TensorFlow Keras on the Kendryte k210, for simple image recognition.
The model has been modified so that it (hopfully) meets all the requirements to use the acceleration of the KPU (changing the padding, reducing the model size).

I can load and run the model (I followed some of the examples).
(https://github.com/kendryte/nncase/blob/master/examples/20classes_yolo/k210/kpu_20classes_example/main.c)
The model is simple and should distinguish only 8 classes, so I am using a simplified version of the example.

Now to my problem: If I use the function "kpu_get_output(...)" and print out the values of the classes, I always get the same values, no matter of the input image.

The input image is not alwayse the same. I have checked that.
The model is poorly trained at the moment, but this should not be a problem as the output values should still change.

If anyone has an idea what the problem could be or what I could try, or, if there is still certain information missing to solve the problem, I would appreciate a comment.

**Comment by MoritzKrauth** [comment]
So I found the problem. I used a flatten layer at the end, but forgot to check if I am still in the 1024 channels range. With this I close the post.

**Comment by WlearnZ** [comment]
how to transform the .tflite model to .kmodel
